.travis.yml

```diff
@@ -92,6 +92,14 @@ jobs:
             stage: Test
             php: 7.2
 
+        -
+            <<: *STANDARD_TEST_JOB
+            stage: Test
+            php: 7.2
+            env: STRICT_TYPES=1
+            before_script:
+                - PHP_CS_FIXER_FUTURE_MODE=1 php php-cs-fixer fix --quiet --rules=blank_line_after_opening_tag,declare_strict_types
```

Comment:
we already do this, and more:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.12/.travis.yml#L126

Reply:
Yeah, but this job is allowed to fail, which is happening since I remember :(

Reply:
then let it not allow to fail on newest stable PHP release, and let allow nightly to fail only because it's unstable PHP, not because we run strict checks there

Reply:
So like this or in separate job or how?

Reply:
almost like this - here is the proper place - 7.2 job, but run all extra rules, not just those randomly picked

Reply:
Done (those weren't random - `declare_strict_types` plus `blank_line_after_opening_tag` to cleanup after it).